pai-management/quick-start/services-configuration.yaml.template

```diff
@@ -117,6 +117,10 @@ prometheus:
   #    smtp_auth_username: user@gmail.com
   #    smtp_auth_password: gmail_password
 
+  # if you want to use key file to login nodes:
+  # 1. create secret with command 'kubectl create secret generic ssh-key-secret --from-file=<secret name>=/path/to/keyfile'
```

Comment:
Please add `kubectl create secret xxx` command in `pai-management/bootstrap/prometheus/start.sh` to create secret instead of requiring user read the comments and create secret.

In that case, you should change `pai-management/bootstrap/prometheus/start.sh` to `pai-management/bootstrap/prometheus/start.sh.template` and add entry `start.sh` in `template-list field` of `pai-management/bootstrap/prometheus/service.yaml`, since not all users need to create that secret.